.vscode/settings.json

```
@@ -63,9 +62,7 @@
   },
 
   // "terminal.integrated.defaultProfile.linux": "bash",
-
   "terminal.integrated.splitCwd": "workspaceRoot",
-
   "terminal.integrated.shellIntegration.enabled": false,
```

all of these
```suggestion
  "terminal.integrated.shellIntegration.enabled": true,
```

why this is set to false escapes me

---

last time i was trying to integrate fish that was brand-new feature that was not quite well done and fish support claimed to work, but didn't seem to in actuality&hellip;